src/Repository/AdherentRepository.php

```diff
@@ -549,4 +549,25 @@ public function countCommitteeSupervisorsByGenderForReferentManagedArea(Adherent
             ->getArrayResult()
         ;
     }
+
+    public function countCommitteeMembersByGenderForReferentManagedArea(Adherent $referent): array
+    {
+        if (!$referent->isReferent()) {
+            throw new \InvalidArgumentException('Adherent must be a referent.');
+        }
+
+        return $this->createQueryBuilder('a', 'a.gender')
```

Comment:
Can you uses explicites words instead of one letter for aliases ?

`return $this->createQueryBuilder('adherent', 'adherent.gender')`